mmyolo/models/necks/yolov8_pafpn.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Union
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+from mmdet.models.backbones.csp_darknet import CSPLayer
+from mmdet.utils import ConfigType, OptMultiConfig
+
+from mmyolo.registry import MODELS
+from .yolov5_pafpn import YOLOv5PAFPN
+from .. import CSPLayerWithTwoConv
+from ..utils import make_divisible, make_round
+# from .base_yolo_neck import BaseYOLONeck
+
+
+@MODELS.register_module()
+class YOLOv8PAFPN(YOLOv5PAFPN):
```

Comment:
missing `build_out_layer`, `build_downsample_layer`, and `build_upsample_layer` in ` YOLOv8PAFPN`